canonical-web-and-design/certification.ubuntu.com
# Global nav and google analytics

- Add the google tag manager snippet
- Add back the global nav (not the latest version, as we'll update to that when we redesign for ubuntu.com/certification)

QA
--

`./run`, go to the homepage, check the global nav works, view source, check you see tag manager code.
